tests/integration/standard/test_udts.py

```diff
@@ -580,7 +580,6 @@ def test_nonprimitive_datatypes(self):
         """
         Test for inserting various types of DATA_TYPE_NON_PRIMITIVE into UDT's
         """
-        raise unittest.SkipTest("Collections are not allowed in UDTs")
```

Comment:
There is a check for protocol v3 in the setUp: https://github.com/kishkaru/python-driver/blob/enable_collections_in_udt/tests/integration/standard/test_udts.py#L41 which applies to all tests under the TypeTests class.